modules/messages/src/lib.rs

```diff
@@ -216,7 +222,6 @@ pub mod pallet {
 
 	#[pallet::pallet]
 	#[pallet::generate_store(pub(super) trait Store)]
-	#[pallet::without_storage_info]
```

Comment:
For the other pallets could we remove `#[pallet::without_storage_info]` and use `#[pallet::unbounded]` on a case by case basis ? I think it would be easier to track the storage variables that are unbounded.

Reply:
TBH I don't know what are side effects of that :) So I'd wait and see how do they solve similar problem in the `paras` pallet (here: https://github.com/paritytech/polkadot/blob/master/runtime/parachains/src/paras/mod.rs#L572)

Reply:
Ok